src/components/CategorySelector.tsx

```diff
@@ -141,13 +141,16 @@ const ScrollingSelector = ({
                         "border-sky-400"
                     )}
                   >
-                    <div className="aspect-w-3 aspect-h-2 max-w-[300px] mx-auto">
-                      <Image
-                        className="rounded-lg object-cover object-center  shadow-lg"
-                        image={item.primaryPhoto}
-                        loading="eager"
-                      />
-                    </div>
+                    {item.primaryPhoto && (
+                      <div className="aspect-w-3 aspect-h-2 max-w-[300px] mx-auto">
+                        <Image
+                          className="rounded-lg object-cover object-center  shadow-lg"
+                          image={item.primaryPhoto}
```

Comment:
null check?

Reply:
Are you asking if the null check is required here? I believe it is...

Reply:
Generally seems like a best practice for our Image component right now because we don't gracefully handle passing null images to the Image component. It will break you site if you do that by accident

Reply:
yeah - I think you should do a null check in case them image or the primary photo is undefined.